### ISSUE TITLE
LOG´s 

### ISSUE THREAD

**Issue by marine1988** [Jonybat/flexget_config#5]
Can u share a log with me please just to see if you have same issues!

**Comment by Jonybat** [comment]
What issues?

**Comment by marine1988** [comment]
the+iratebay.se gives me that some times

`2016-09-23 11:42
INFO
trakt_list
update-series-list
Successfully added to/from list series: 0 movie(s), 0 show(s), 0 episode(s), 0 season(s).
2016-09-23 11:46
INFO
manager
Got SIGTERM. Shutting down.
2016-09-23 11:47
INFO
scheduler
Starting scheduler
2016-09-23 11:47
INFO
web_server
Web interface available at http://127.0.1.1:3539
2016-09-23 12:00
INFO
task
download-series-discover
Plugin next_series_episodes has requested task to be ran again after execution has completed. Reason: Look for next season
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: HTTPSConnectionPool(host='thepiratebay.org', port=443): Read timed out. (read timeout=30)
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.
2016-09-23 12:01
ERROR
discover
download-series-discover
Error searching with piratebay: RequestException: Requests to this site (thepiratebay.se) have timed out recently. Waiting before trying again.`

**Comment by Jonybat** [comment]
Everything works fine here. Looks like connectivity issues or isp blocking

**Comment by marine1988** [comment]
no isp blocking and no network issues. i think is timeout from piratebay

**Comment by marine1988** [comment]
![image](https://cloud.githubusercontent.com/assets/6524068/18784140/200bd51c-8188-11e6-8ac5-76b33d7f2e99.png)

**Comment by Jonybat** [comment]
Don't know what to tell you. Tpb is working fine here

**Comment by marine1988** [comment]
can you share your log for me to compare please?

**Comment by Jonybat** [comment]
Ok, i think i figured out what is the issue here. I added more items to search and it started timing out after 10 series (which i think relates to 2/3 searches per episode). There seems to be some kind of flood protection system that limits the amount of requests per a certain amount of time (haven't figured how much yet). Once it reaches that limit, all the requests will fail, until it resets the counter.

I increased the delay between requests from 1s to 5s (wasn't enough) and finally to 10s. I was able to search 20 episodes without timing out.

See: https://github.com/Jonybat/flexget_config/commit/2af2d7487e7ad6ad90632330908f2f5cdceb27f8
